src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -28,13 +29,15 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + PREFIX_DRUG_ALLERGY + "DRUG ALLERGIES "
+            + PREFIX_GENDER + "GENDER"
```

Comment:
```suggestion
            + PREFIX_GENDER + "GENDER "
```
space to make things look neater